proto/block.proto

```diff
@@ -23,8 +23,10 @@ message BlockHeader {
     digest.Digest tx_hash = 8;
     // a hash of a STARK proof attesting to the correct state transition.
     digest.Digest proof_hash = 9;
+    // an accumulative hash from all kernel hashes.
+    digest.Digest kernel_root = 10;
```

Comment:
nit: I would change this to: "a commitment to all transaction kernels supported by this block."

Reply:
Changed. Maybe we want to change it on `miden-base` as well?

Reply:
Good idea! Done in https://github.com/0xPolygonMiden/miden-base/pull/850/commits/a9938cd0cf3480ea8371393f250c36807404d18d.